lib/galaxy/tools/parameters/basic.py

```diff
@@ -156,6 +155,11 @@ def value_from_basic( self, value, app, ignore_errors=False ):
             return self.to_python( value, app )
 
     def value_to_display_text( self, value, app=None ):
+        if is_runtime_value( value ):
+            return "Not available."
+        return self.to_text( value, app )
+
+    def to_text( self, value, app ):
```

Comment:
You can remove `app` from the list of paramaters, it's not used either here or in the overridden methods.